tsc errors when bumping to `@opentelemetry/auto-instrumentations-node@0.67.0`

### What version of OpenTelemetry are you using?

Attempting to update to `@opentelemetry/auto-instrumentations-node@0.67.0`

### What version of Node are you using?

`v22.14.0`

### What did you do?

Bump to `@opentelemetry/auto-instrumentation-node` at `^0.67.0`, then run `tsc --skipLibCheck false --incremental false`

### What did you expect to see?

Project should compile without errors.

### What did you see instead?

```
node_modules/@opentelemetry/instrumentation-openai/build/src/instrumentation.d.ts:3:86 - error TS2307: Cannot find module 'openai/resources/chat/completions' or its corresponding type declarations.

3 import type { ChatCompletion, ChatCompletionCreateParams, ChatCompletionChunk } from 'openai/resources/chat/completions';
                                                                                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@opentelemetry/instrumentation-openai/build/src/instrumentation.d.ts:4:69 - error TS2307: Cannot find module 'openai/resources/embeddings' or its corresponding type declarations.

4 import type { CreateEmbeddingResponse, EmbeddingCreateParams } from 'openai/resources/embeddings';
                                                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Found 2 errors in the same file, starting at: node_modules/@opentelemetry/instrumentation-openai/build/src/instrumentation.d.ts:3
```

### Additional context

If I run `yarn why openai`, I do not show that as installed in my project. Perhaps `openai` should be a `dependency` instead of a `devDependency` in `@opentelemetry/instrumentation-openai`?

<sub>**Tip**: [React](https://github.blog/news-insights/product-news/add-reactions-to-pull-requests-issues-and-comments/) with 👍 to help prioritize this issue. Please use comments to provide useful context, avoiding `+1` or `me too`, to help us triage it. Learn more [here](https://opentelemetry.io/community/end-user/issue-participation/).</sub>

--- a/packages/instrumentation-openai/src/instrumentation.ts
+++ b/packages/instrumentation-openai/src/instrumentation.ts
@@ -169,7 +169,7 @@ export class OpenAIInstrumentation extends InstrumentationBase<OpenAIInstrumenta
     );
   }
 
-  _getPatchedChatCompletionsCreate(): any {
+  private _getPatchedChatCompletionsCreate(): any {
     const self = this;
     return (original: ChatCompletions['create']) => {
       // https://platform.openai.com/docs/api-reference/chat/create
@@ -251,7 +251,7 @@ export class OpenAIInstrumentation extends InstrumentationBase<OpenAIInstrumenta
    * Start a span for this chat-completion API call. This also emits log events
    * as appropriate for the request params.
    */
-  _startChatCompletionsSpan(
+  private _startChatCompletionsSpan(
     params: ChatCompletionCreateParams,
     config: OpenAIInstrumentationConfig,
     baseURL: string | undefined
@@ -449,7 +449,7 @@ export class OpenAIInstrumentation extends InstrumentationBase<OpenAIInstrumenta
    * async iterator. It should yield the chunks unchanged, and gather telemetry
    * data from those chunks, then end the span.
    */
-  async *_onChatCompletionsStreamIterator(
+  private async *_onChatCompletionsStreamIterator(
     streamIter: AsyncIterator<ChatCompletionChunk>,
     span: Span,
     startNow: number,
@@ -604,7 +604,7 @@ export class OpenAIInstrumentation extends InstrumentationBase<OpenAIInstrumenta
     span.end();
   }
 
-  _onChatCompletionsCreateResult(
+  private _onChatCompletionsCreateResult(
     span: Span,
     startNow: number,
     commonAttrs: Attributes,
@@ -702,7 +702,7 @@ export class OpenAIInstrumentation extends InstrumentationBase<OpenAIInstrumenta
     span.end();
   }
 
-  _createAPIPromiseRejectionHandler(
+  private _createAPIPromiseRejectionHandler(
     startNow: number,
     span: Span,
     commonAttrs: Attributes
@@ -734,7 +734,7 @@ export class OpenAIInstrumentation extends InstrumentationBase<OpenAIInstrumenta
     };
   }
 
-  _getPatchedEmbeddingsCreate(): any {
+  private _getPatchedEmbeddingsCreate(): any {
     const self = this;
     return (original: Embeddings['create']) => {
       // https://platform.openai.com/docs/api-reference/embeddings/create
@@ -778,7 +778,7 @@ export class OpenAIInstrumentation extends InstrumentationBase<OpenAIInstrumenta
    * Start a span for this chat-completion API call. This also emits log events
    * as appropriate for the request params.
    */
-  _startEmbeddingsSpan(
+  private _startEmbeddingsSpan(
     params: EmbeddingCreateParams,
     baseURL: string | undefined
   ) {
@@ -810,7 +810,7 @@ export class OpenAIInstrumentation extends InstrumentationBase<OpenAIInstrumenta
     return { span, ctx, commonAttrs };
   }
 
-  _onEmbeddingsCreateResult(
+  private _onEmbeddingsCreateResult(
     span: Span,
     startNow: number,
     commonAttrs: Attributes,

